psr/with-let-letting.tex

```diff
@@ -0,0 +1,233 @@
+
+This PSR adds defintions for \gn{let\_clause} and \gn{letting\_clause} to figure 3 of the PartiQL specification 
```

Comment:
Not sure if this is necessarily part of the PSR, but where will the `WITH`, `LET`, and `LETTING` sections be added to the spec?

Reply:
This is mostly a new section--I think we can leave that as a decision for later, when the PSR has been approved by the spec committee.